flink-clients/src/main/java/org/apache/flink/client/LocalExecutor.java

```diff
@@ -146,21 +129,9 @@ private JobExecutorService createJobExecutorService(Configuration configuration)
 		return miniCluster;
 	}
 
-	@Override
-	public void stop() throws Exception {
-		synchronized (lock) {
-			if (jobExecutorService != null) {
-				jobExecutorService.close();
-				jobExecutorService = null;
-			}
-		}
-	}
-
-	@Override
-	public boolean isRunning() {
-		synchronized (lock) {
-			return jobExecutorService != null;
-		}
+	private void stopExecutorService(final JobExecutorService executorService) throws Exception {
```

Comment:
I'm highly suspect the value brought by `JobExecutorService`. Its only implementation is `MiniCluster`. Also, it might have name conflict with the ongoing "Executor".

Reply:
I am not so familiar with this code and maybe @tillrohrmann has more to say on the value of the `JobExecutorService`. In any case, I believe this is not related to this PR and can be added in the future. What do you think?

Reply:
The `JobExecutor` and the `JobExecutorService` have been introduced to bridge between the Flip-6 `MiniCluster` and the legacy `FlinkMiniCluster`. They should be obsolete now and could be removed if needed.

Reply:
Thanks for you information @tillrohrmann. @kl0u I'd like to make this removal a separated issue, fyi FLINK-13961.

Reply:
Sounds good @TisonKun .